src/index.js

```diff
@@ -68,11 +48,7 @@ function bindEvents (options) {
     documentCapture: () => options.onDocumentCapture(getCaptures().documentCapture),
     documentBackCapture: () => options.onDocumentCapture(getCaptures().documentBackCapture),
     faceCapture: () => options.onFaceCapture(getCaptures().faceCapture),
-    complete: () => options.onComplete(getCaptures()),
-    onError: () => {
```

Comment:
do you guys think it's relevant to have an error callback for the developer? @hjheath @stefaniacardenas